frontend/src/components/StockDetails/StockQuote/StockQuote.js

```diff
@@ -34,36 +37,31 @@ class StockQuote extends React.Component {
         return format(date, "MMM dd, yyyy");
     }
     render() {
+        const { c, ...props } = this.state.stockQuote;
```

Comment:
Change `...props` to `quoteProps` for clearer naming.

Reply:
Renamed to quoteProps, ... for spreading properties. Let me know if you would prefer having each property individually listed instead of spreaded.

Reply:
I prefer spreaded.